src/applications/edu-benefits/tests/edu-apply-wizard.cypress.spec.js

```diff
@@ -0,0 +1,142 @@
+// const E2eHelpers = require('../../../platform/testing/e2e/helpers');
+// const Timeouts = require('../../../platform/testing/e2e/timeouts.js');
+
+describe('Education Application Wizard', () => {
+  it('Goes through the education wizard', () => {
+    // Ensure education apply-wizard page renders.
+    // Open education apply wizard
+    cy.visit(`/education/how-to-apply/`)
+      .get('body', { timeout: 10000 })
```

Comment:
Is ten seconds needed here and on the next few? Seems high. Do we want to define a constant for like a normal and/or long timeout? And use those throughout?

Reply:
updated to 1000 milliseconds